assets/base/i18n/en.json

```diff
@@ -10,6 +10,26 @@
   "about.teamEditiont0": "Team Edition",
   "about.teamEditiont1": "Enterprise Edition",
   "about.title": "About {appTitle}",
+  "accessibility.attachment_button": "attachment button",
+  "accessibility.channel_nav_bar": "channel nav bar",
+  "accessibility.channel_post_list": "channel post list screen",
+  "accessibility.login": "login screen",
+  "accessibility.login_options": "login options screen",
+  "accessibility.login_options.email_button": "email login option button",
+  "accessibility.login_options.gitlab_button": "gitlab login option button",
+  "accessibility.login_options.logo_image": "logo image",
+  "accessibility.login.email_or_username input": "",
+  "accessibility.login.forgot_password_button": "forgot password button",
+  "accessibility.login.logo_image": "logo image",
+  "accessibility.login.password_input": "password input",
+  "accessibility.login.sign_in_button": "sign in button",
+  "accessibility.post_textbox": "post textbox",
+  "accessibility.post_textbox.text_input": "text input",
+  "accessibility.select_server": "select server screen",
+  "accessibility.select_server.connect_button": "connect button",
+  "accessibility.select_server.logo_image": "logo image",
+  "accessibility.select_server.url_input": "url input",
+  "accessibility.send_button": "send button",
```

Comment:
@esethna can you review the text for this accessibility labels? Thanks

Reply:
Why  are we adding accessibility labels in this PR? Seems like it should be done separately?

Reply:
IIRC The e2e framework uses it to identify the elements on the screen, so in order to have e2e tests these are needed

Reply:
Accessibility label or ID is the primary suggested [locator strategy](http://appium.io/docs/en/commands/element/find-elements/) for Appium automation, with the ff reasons:
- works the same for both iOS and Android
- fastest locator strategy compared to others since it will go straight to accessibility labels other than walking to the entire tree/view
- fast and less flaky since it will look for element inside the viewing window, and not those outside. Test on element is mostly on how the user sees the UI.